### PR TITLE
Changes to documentation section with new server card.

### PR DESCRIPTION
**Pull request by zorkow**
Changes as discussed in our f2f:

* Getting started button in the header.
* Documentation button now points to docs pages.
* Documentation section is now getting started section.
* Added a card for server integration.
* Improved the back-linking from the close buttons to the cards.

### CODE REVIEW

File: _includes/frontpage/docs.html [hunk 1]
```diff
@@ -1,39 +1,59 @@
...
+      </img>
+    <a href="https://www.npmjs.com/package/mathjax-full" class="shield">
+      <img class="shield" alt="MathJax sources npm package"
+           src="https://img.shields.io/npm/v/mathjax-full?label=npm@sources">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax-node" class="shield">
+      <img class="shield" alt="MathJax version 2 legacy npm package"
+           src="https://img.shields.io/npm/v/mathjax-node?label=npm@legacy-v2">
+      </img>
```
Comment (dpvc) [inline _includes/frontpage/docs.html]: I would not list `mathjax-node`, at least not as `npm@legacy-v2`.  I thought this was indicating the latest v2 version on npm, as the shield doesn't make any indication that this is about `mathjax-node`.

File: _includes/frontpage/docs.html [hunk 2]
```diff
@@ -1,39 +1,59 @@
...
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax/legacy-v2">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax" class="shield">
+      <img class="shield" alt="MathJax npm package"
+           src="https://img.shields.io/npm/v/mathjax">
+      </img>
+    <a href="https://www.npmjs.com/package/mathjax-full" class="shield">
+      <img class="shield" alt="MathJax sources npm package"
+           src="https://img.shields.io/npm/v/mathjax-full?label=npm@sources">
```
Comment (dpvc) [inline _includes/frontpage/docs.html]: I'm wondering if `npm@full` is easier to tie to the `mathjax-full` package than `npm@source`?

File: _includes/frontpage/docs.html [hunk 3]
```diff
@@ -1,39 +1,59 @@
...
-          <h3>Getting Started</h3>
+          <h3>Web Integration</h3>
           <p>
             <img src="img/code.svg" alt="" class="card--img">
             Whether you're a casual user, a serious author, or a professional developer, it's easy to
             integrate MathJax.
           </p>
         </div>
-        <a class="btn" href="#gettingstarted" id="to-gettingstarted">Learn more</a>
+        <a class="btn" href="#gettingstarted">Start now</a>
```
Comment (dpvc) [inline _includes/frontpage/docs.html]: I agree that the bounce-back is not great, but if you are going to make a change like this, it should be done to all the buttons in all the section, not just this one.  

It turns out that there is a script that modifies the targets of the back links and it is controlled by an array in the `main.js` file:

https://github.com/mathjax/MathJax-website/blob/f2f8fa592a41e8ae09f1342ffb0559573c6fff93/js/main.js#L67-L100

You need to add `#to-intsallnow` to that list to get consistent behavior for the new card.

If you want to change the bound-back to not use this approach (it was not my doing), and change all the links, I'm good with that, but the current setup is inconsistent from section to section, and for this new card compared to the other cards.

File: _includes/header.html [hunk 4]
```diff
@@ -2,7 +2,8 @@
     <ul>
         <li><a href="{{site.baseurl}}/#services">Services</a></li>
         <li><a href="{{site.baseurl}}/#features">Features</a></li>
-        <li><a href="{{site.baseurl}}/#docs">Documentation</a></li>
+        <li><a href="{{site.baseurl}}/#docs">Getting Started</a></li>
+        <li><a href="http://docs.mathjax.org/">Documentation</a></li>
```
Comment (dpvc) [inline _includes/header.html]: If you change this to `https` you will not get the security warnings that you mentioned f2f.

There are a bunch of other links to the docs that are also `http:` in `_includes/frontpage/docs.html`.  Also, all the sponsor links seem to be `http:`, so perhaps those should go to `https:` as well.

**Review by dpvc — CHANGES_REQUESTED** [review]
This looks good over all.  Thanks for doing it.  I did request a couple of adjustments.

File: _includes/frontpage/docs.html (hunk 2 above)
Comment (zorkow) [reply _includes/frontpage/docs.html]: I've changed it to `npm@full`. But what we really want is something to indicate that this is for developers, whereas `npm` is for the casual user. Not sure how to write that.

File: _includes/frontpage/docs.html (hunk 1 above)
Comment (zorkow) [reply _includes/frontpage/docs.html]: Removed the shield.

File: _includes/header.html (hunk 4 above)
Comment (zorkow) [reply _includes/header.html]: The issue here is that the `docs.mathjax.org` site is not secure. I can fix up all links to `https` but they might fall back to `http`.

Comment (zorkow) [reply _includes/header.html]: Correction: `https://docs.mathjax.org` actually works, but for some reason the link from `mathjax.org` still falls back to `http`. Might be my jekyll setup, though.

File: _includes/frontpage/docs.html (hunk 3 above)
Comment (zorkow) [reply _includes/frontpage/docs.html]: I've updated the table so all cards have the same original behaviour.

**Comment by zorkow** [comment]
Included all suggestions. PTAL.
I will do security updates in a separate PR.

**Review by dpvc — APPROVED** [review]

**Pull request merged**